pkg/config/config.go

```diff
@@ -116,11 +116,14 @@ type DisabledChecks struct {
 // NewConfig initializes a new config
 func NewConfig(configPaths []string) (*Config, error) {
 	if len(configPaths) == 0 {
-		return nil, fmt.Errorf("No config paths provided")
+		return nil, fmt.Errorf("no config paths provided")
 	}
 
 	var config Config
 
+	// initialize the Logger early, so usage of it does not fail
+	config.Logger.Init()
```

Comment:
I would have added a Init to config, that would have called the Logger Init

This way, if the logger changes nothing needs to be updated outside Config

But why not, it's the NewConfig constructor

Reply:
I later switched to a lazy initialization, which is more reliable, so this was not needed at all. I removed it.